posts/how-to-efficiently-use-jhipster-in-your-company.md

```diff
@@ -0,0 +1,271 @@
+---
+authors:
+- Theo Lebrun
+categories:
+- JHipster
+date: 2018-03-23T16:12:10.000Z
+title: "How to efficiently use JHipster in your company"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/03/jhipster-module-header.png
+---
+
+[JHipster](http://www.jhipster.tech/) is a fantastic development platform if you want to generate, develop and deploy Spring Boot + Angular Web applications. I personnally use it to generate multiple Spring microservices that will be already configured to work in my company's infrastructure. The goal of this blog post is to show how you can easily customize JHipster so it will fit your company's requirements and infrastructure. A common use case after generating a microservice is to add common dependencies, extra configuration or change the Jenkinsfile/Dockerfile. The idea is to include all those changes in a JHipster module to avoid spending time repeating those extra steps. And to make things even more easier, I will detail a way to avoid having to answer the questions that JHipster ask during the generation.
```

Comment:
"...that will be already configured..." -> "...that are preconfigured..."
"...even more easier..." -> "...even easier..."
"ask" -> "asks"